Sprint _ 2

akimov058 opened the pull request.

Commit:
Добавлен тест test_get_books_for_children_neg на проверку, что книги с возврастным ограничением не возвращаются в списке

Commit:
Добавлен тест test_add_book_in_favorites, который добавляет книгу в избранное и проверяет её по названию

Commit:
Добавлен тест test_add_book_in_favorites_add_two_equal_books, который пытается два раза добавить одну книгу в избранное

Commit:
Добавил тест test_delete_book_from_favorites и мелкие правки

Commit:
правки

Commit:
Добавил кейс test_get_books_with_specific_genre

Commit:
Добавил кейс test_get_book_genre и Заполнил REAME

AlexKitIt commented on tests.py:
```diff
@@ -1,24 +1,114 @@
+        collector = BooksCollector()
+        collector.add_new_book(book_name)
+        collector.set_book_genre(book_name,'Фантастика')
+        assert collector.get_books_genre()==collector.books_genre
```
Нужно исправить: проверка этого метода должна осуществляться по конкретному словарю

akimov058 replied:
Добрый день, спасибо за замечание
Исправил замечание, сделал Push

Commit:
Исправил кейс test_get_books_genre, учёл замечание ревьювера
AlexKitIt 9 hours ago
Нужно исправить: проверка этого метода должна осуществляться по конкретному словарю
